src/plotly-graph-card.ts

```diff
@@ -432,6 +456,7 @@ export class PlotlyGraph extends HTMLElement {
             throw new Error(
               `period: "${entity.period}" is not valid. Use ${STATISTIC_PERIODS}`
             );
+          entity.extend_to_present ??= !!entity.statistic;
```

Comment:
This condition should be flipped. Now `extend_to_present` is true when using statistics.

Reply:
Woho, good catch